nodes/nodes/nodes-nodes-rebooting.adoc

```diff
@@ -1,35 +1,37 @@
 
 :context: nodes-nodes-rebooting
 [id="nodes-nodes-rebooting"]
-= Understanding node rebooting
+= Managing node rebooting
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 
 
-To reboot a node without causing an outage for applications running on the
-platform, it is important to first evacuate the pods. For pods that are 
-made highly available by the routing tier, nothing
-else needs to be done. For other pods needing storage, typically databases, it
-is critical to ensure that they can remain in operation with one pod
-temporarily going offline. While implementing resiliency for stateful pods
-is different for each application, in all cases it is important to configure
-the scheduler to use node anti-affinity to
+There is no {product-title} command to reboot a node. However, if you need to manually reboot one of your {product-title} cluster nodes,
+there are some issues you need to consider. 
+
+In order to reboot a node without causing an outage for applications running 
+on the platform, it is important to first evacuate the pods. For pods that are 
+made highly available by the routing tier, nothing else needs to be done. 
+
+For other pods needing storage, typically databases, it is critical to ensure 
+that they can remain in operation with one pod temporarily going offline. 
+While implementing resiliency for stateful pods is different for each application, 
+in all cases it is important to configure the scheduler to use 
+xref:../../nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-affinity_nodes-nodes-rebooting[node anti-affinity] to
 ensure that the pods are properly spread across available nodes.
 
-Another challenge is how to handle nodes that are running critical
-infrastructure such as the router or the registry. The same node evacuation
-process applies, though it is important to understand certain edge cases.
+Another challenge is how to handle xref:../../nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-infrastructure_nodes-nodes-rebooting[rebooting master nodes], which running critical infrastructure such as a router or the registry. The same node evacuation process applies, though it is important to xref:../../nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-router_nodes-nodes-rebooting[understand certain edge cases].
```

Comment:
We explicitly exclude the router pods from control-plane nodes [by default][1] because of an upstream bug (only [recently fixed][2]).  Should we consolidate here to just talk about the registry?  I'd expect the registry operator to have the appropriate pod disruption budgets in place already, and not be something we needed to walk folks through in docs.

[1]: https://github.com/openshift/cluster-ingress-operator/blob/73d6fd5de8061fa21b89ca0a26546697fff3fde5/manifests/00-custom-resource-definition.yaml#L308-L312
[2]: https://github.com/kubernetes/kubernetes/issues/65618#issuecomment-628902088